lib/mix/tasks/appsignal.demo.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Mix.Tasks.Appsignal.Demo do
```

Comment:
Could you rename this file to `lib/mix/tasks/appsignal/demo.ex`? That's more according to the conventions.

Oh wait, nevermind! Looking at Phoenix I see they do it also like this